vault-password-policies/intro.md

```diff
@@ -0,0 +1,9 @@
+Vault can gen onfigurable password policies today and the impetus as to why we
+decided to do this. So, as you may know vault can generate dynamic usernames and
+passwords and we do that by randomly. Picking characters from a base 62
+character set right so when we decided to roll this out to certain customers.
+They came back and said, hey, this password that you're generating is to secure
+That means our older systems that we're connecting to like DB two systems have a
+very strict password character SEC requirement, they need A character password
+length of eight characters and has to be one uppercase one lowercase and one
+number one character only
```

Comment:
This intro text does not appear to be complete yet?

Reply:
Yes, pushing the change now that the guide has gotten a review.